examples/cloudformationtemplates/s3bucket.yaml

```diff
@@ -122,18 +145,40 @@ data:
               Value: !Ref ClusterName
             - Key: Heritage
               Value: operator.aws
-          VersioningConfiguration: !If 
+          VersioningConfiguration: !If
             - UseVersioning
             - Status: Enabled
             - !Ref 'AWS::NoValue'
         DeletionPolicy: Retain
+
+      WebsiteBucketPolicy:
+        Type: AWS::S3::BucketPolicy
+        Condition: UseAsStaticSite
+        Properties:
+          Bucket: !Ref S3bucket
+          PolicyDocument:
+            Statement:
+            -
+              Action:
+              - "s3:GetObject"
+              Effect: Allow
+              Principal: "*"
+              Resource:
+                Fn::Join:
```

Comment:
MINOR: for consistency I like to stick with either `!Join` for `Fn::Join` styles of calling intrinsic functions